tests/plugins/balloontoolbar/plugin.js

```diff
@@ -0,0 +1,26 @@
+/* bender-tags: balloontoolbar, 1346, context */
+/* bender-include: _helpers/default.js */
+/* global ignoreUnsupportedEnvironment */
+
+( function() {
+	'use strict';
+
+	bender.editor = {};
+
+	var tests = {
+		'test context api is available in plugin.init()': function() {
+			CKEDITOR.plugins.add( 'testplugin', {
+				requires: 'balloontoolbar',
+				init: function( editor ) {
+					assert.isNotUndefined( editor.balloonToolbars );
+				}
+			} );
+
+			bender.editorBot.create(
+			{ name: 'editor2', config: { extraPlugins: 'testplugin' } }, function noop(  ) {} );
```

Comment:
This line should be indented (or you may simply put everything in the one line). Also I don't see any benefit in naming this `noop` function. You may simply use `bender.editorBot.create( { ... }, function() {} );`.